hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -3444,6 +3444,16 @@
     </description>
   </property>
 
+  <property>
+    <name>dfs.client.lease.renewal.interval.ms</name>
+    <value>0</value>
+    <description>
+      If set between 0 and 30000 inclusive, HDFS clients will renew leases for files they are writing at this interval.
+      If dfs.client.lease.renewal.interval.ms is not set and ipc.client.rpc-timeout.ms is set between 0 and 60000,
```

Comment:
What is magic about the `60000` maximum value of `ipc.client.rpc-timeout.ms` ? I guess that comes from existing behavior?

Reply:
It comes from existing behavior, yes. The lease renewal interval isn't allowed to be longer than half of the NameNode's timeout, which is hard-coded at 60 seconds.